requirements/common.in

```diff
@@ -193,3 +193,6 @@ circuitbreaker
 
 # Runtime monkeypatching of django-stubs generics
 django-stubs-ext
+
+# Needed for preferred HTTP responses
+accept-types 
```

Comment:
It's great that you figured out how to add new dependencies, but I'm really not convinced this adds much.

You should write your own function which takes a potential list of content-types and the request.  It should pull out the list of [`MediaType` objects](https://github.com/django/django/blob/32d4b61c313be5169137047e9fb3668da20a5d89/django/http/request.py#L663-L691) which are [returned from calling `request.accepted_types`](https://github.com/django/django/blob/32d4b61c313be5169137047e9fb3668da20a5d89/django/http/request.py#L107-L109), which represent what the client accepts, and their weightings.

Sort the types which are acceptable to the client by weight (as determined by accessing `.params.get("q", 1.0)`), and then for each of those, return the first of the server content-types which matches (judged using `.match(...)`).